tools/bin/commands/build

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+set -eu
```

Comment:
normally such kind of global settings should be dealt with in the top level "syndesis" script (where there is already a set -eu https://github.com/syndesisio/syndesis/blob/0420a964654c8ac448476f394858234ae0ca61d9/tools/bin/syndesis#L20-L21

Modules ideally should only hold functions.